platform/security/core/security-core-impl/src/main/java/ddf/security/service/impl/SecurityAssertionStore.java

```diff
@@ -48,26 +47,24 @@
     public static SecurityAssertion getSecurityAssertion(Message message) {
         if (message != null) {
             TokenStore tokenStore = getTokenStore(message);
-            Principal principal = (Principal) message.get(WSS4JInInterceptor.PRINCIPAL_RESULT);
```

Comment:
What happened to this? This is where the principal ends up after it rolls through the WSS4J interceptors.